new_django_project/blog/api/models.py

```diff
@@ -9,3 +9,12 @@ class Post(models.Model):
     class Meta:
         ordering = ['created']
 
+class Comment(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    body = models.TextField(blank=False)
+    owner = models.ForeignKey('auth.User', related_name='comments', on_delete=models.CASCADE)
+    post = models.ForeignKey('Post', related_name='comments', on_delete=models.CASCADE)
```

Comment:
@helenamagaldi do you know what the `CASCADE` method at `on_delete` do? Is this the wanted behavior? (I think it makes sense here! It's just a double check because as it is the default method it is commonly used without a lot of thought and it brings problems if we want to keep the data even when the other object is deleted 😱 )

Reply:
Yeah, I do. I just figured it was the method that made the most sense in order to achieve the required functionality. 

But I got a bit confused with these problems you've mentioned. Would you care to elaborate a little bit further, please? <3

Reply:
Sure! If this is the required behavior for this functionality it will not be a problem at all 😄  
It would be a problem if you wanted to keep the comments regarding the deletion of a post or an owner (idk, sometimes you want to keep a comment in a post even if the owner was deleted and instead of deleting the comment just change its owner to "anonymous" or something (?)) and in **this case** the `CASCADE` method would be a problem because it would delete the comment the moment the other objects were deleted and not give you the option to keep it.

Reply:
But it really makes sense to use `CASCADE` here, it was just a double check as I didn't know the desired behavior ;D